bundle/regal/util/util.rego

```diff
@@ -25,8 +25,8 @@ find_duplicates(arr) := {indices |
 
 # METADATA
 # description: returns true if array has duplicates of item
-has_duplicates(array, item) if count([x |
-	some x in array
+has_duplicates(arr, item) if count([x |
```

Comment:
Nice side effect of args counting as vars is how this now got flagged as shadowing the built-in `array` namespace.